modules-js/react-fleet/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { css } from 'emotion';
+
+import { CHARLES_BLUE, OPTIMISTIC_BLUE } from '../utilities/constants';
+
+interface Props {
+  steps: string[];
```

Comment:
I _personally_ don’t mind using an array of strings like this, but can certainly understand why someone would prefer to avoid. I’ll make that change, and change the `showStepName` property so it accepts a string for the task’s name instead.

Reply:
Thanks!